apps/healthlocker/web/router.ex

```diff
@@ -70,8 +70,8 @@ defmodule Healthlocker.Router do
 
     resources "/users", UserController do
       resources "/medication", MedicationController, only: [:index, :show]
+      resources "/about-me", AboutMeController, only: [:new, :edit, :create, :update]
```

Comment:
The nested rousources (https://hexdocs.pm/phoenix/routing.html#nested-resources) works here.
Maybe we can see later on if a nested scope (https://hexdocs.pm/phoenix/routing.html#scoped-routes) is a better representation of the endpoint "about-me" as the relation between user and about-me is one-to-one.